guides/common/modules/proc_configuring-capsule-default-certificate.adoc

```diff
@@ -59,7 +58,7 @@ _output omitted_
 foreman-installer \
 --scenario foreman-proxy-content \
```

Comment:
This should really use `{installer-scenario-smartproxy}` 
```suggestion
{installer-scenario-smartproxy} \
```

Then it can probably be merged with the capsule command.